src/test/java/domain/card/DeckTest.java

```diff
@@ -13,14 +14,21 @@
 
     @BeforeEach
     void setUp() {
-        deck = Deck.initDeck();
+        deck = Deck.initDeck(Card.values());
     }
 
     @Test
-    @DisplayName("Deck을 초기화했을 때, 카드 수가 맞게 초기화되는지 테스트")
+    @DisplayName("Deck을 초기화했을 때, 카드가 모두 덱에 저장되는지 테스트")
     public void initDeckTest() {
-        int expected = 52;
-        assertThat(deck.getCards().size()).isEqualTo(expected);
+        Card card1 = Card.valueOf(Symbol.DIAMOND, Denomination.TEN);
+        Card card2 = Card.valueOf(Symbol.DIAMOND, Denomination.NINE);
+        Card card3 = Card.valueOf(Symbol.DIAMOND, Denomination.SEVEN);
+
+        Deck testDeck = Deck.initDeck(Arrays.asList(card1, card2, card3));
+        List<Card> deckCards = testDeck.getCards();
+        assertThat(deckCards).contains(card1);
+        assertThat(deckCards).contains(card2);
+        assertThat(deckCards).contains(card3);
```

Comment:
```suggestion
    assertThat(deckCards).containsAll(Arrays.asList(card1,card2,card3));
```
`containsAll` 메서드도 있어요 😃

Reply:
AssertJ에 유용한 메서드가 정말 많네요~~ 세번 작성한 것 뭔가 민망하네요 🥲